packages/amplify-e2e-tests/tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "outDir": "lib",
     "lib": ["es2015", "es2016.array.include", "esnext.asynciterable", "dom"],
     "declaration": true,
-    "typeRoots": ["../../node_modules/@types", "node_modules/@types", "./typings"]
+    "typeRoots": ["../../node_modules/@types", "node_modules/@types", "./typings"],
+    "resolveJsonModule": true
```

Comment:
Is this config needed? I see it is required if you import directly from a JSON file [here](https://www.typescriptlang.org/tsconfig#resolveJsonModule). In the changes for cleanup script, I see they are parsed by a readFileSync and JSON.parse

Reply:
Removed, not needed